dependencies.yaml

```
@@ -0,0 +1,84 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - py_version
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+      - test_cpp 
+  checks:
+    output: none
+    includes:
+      - build
+      - develop
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cudf=22.12.*
+          - cmake>=3.23.1
+          - cython>=0.29,<0.30
+          - scikit-build>=0.13.1
+          - gdal>3.5.0,<3.6.0
+          - geopandas>=0.11.0
+  cudatoolkit:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - cudatoolkit=11.5
+  develop:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - pre-commit 
+  doc:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - pydata-sphinx-theme
+          - myst-parser
+  py_version:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+            packages:
+              - python>=3.8,<3.10
```

Use a specific Python version if requested in the matrix, and fall back to the broader pinning of `python>=3.8,<3.10` if `py` is not specified in the matrix.
```suggestion
  py_version:
    specific:
      - output_types: conda
        matrices:
          - matrix:
              py: "3.8"
            packages:
              - python=3.8
          - matrix:
              py: "3.9"
            packages:
              - python=3.9
          - matrix:
            packages:
              - python>=3.8,<3.10
```